src/lua/functions/core/game/config_functions.cpp

```diff
@@ -101,6 +101,7 @@ void ConfigFunctions::init(lua_State* L) {
 	registerEnumIn(L, "configKeys", RATE_MAGIC);
 	registerEnumIn(L, "configKeys", RATE_SPAWN);
 	registerEnumIn(L, "configKeys", HOUSE_PRICE);
+	registerEnumIn(L, "configKeys", HOUSE_BUY_LEVEL)
```

Comment:
I've been already sent this.

Reply:
https://github.com/opentibiabr/canary/blob/main/src/lua/functions/core/game/config_functions.cpp#L103

Reply:
![image](https://user-images.githubusercontent.com/82473579/218486877-d8a82c80-7b01-4265-930e-97a0591fc442.png)

Reply:
![image](https://user-images.githubusercontent.com/82473579/218488153-f92298bb-5048-4066-801b-7282d8393a7d.png)

Reply:
https://github.com/opentibiabr/canary/pull/814/files#diff-84aedd90978a6657254fad1bfc00a15296eceb818b3f538a685aa9a11fff2ea4